src/main/resources/public/error/404.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
I added these folders and files to give you a simple error 404 page and remove the whitespace page. You can add other error pages by just making a different (error number).html page.

Reply:
Nice tip!